packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -143,34 +243,287 @@ class FlutterErrorDetails {
 
   @override
   String toString() {
-    final StringBuffer buffer = StringBuffer();
-    if ((library != null && library != '') || (context != null && context != '')) {
+    // TODO(jacobr): this logic has a fair amount of overlap with
```

Comment:
@Hixie Any thoughts on whether the divergence between this method and the output that is dumped to the console is useful or not? I've matched the display of the old method but I'm lacking the context to understand whether these differences are useful or historic.